CPP/Algorithm/LCA.cpp

```diff
@@ -1,112 +0,0 @@
-/*input
```

Comment:
Is there any issue with this file? why did you delete this file? You didn't mention the reason also you didn't send any replacement for the same